### PR TITLE
Fix: typo using bt2.ClockClassOffset constructor

### PR DESCRIPTION
**Pull request by frdeso**
Signed-off-by: Francis Deslauriers <francis.deslauriers@efficios.com>

### CODE REVIEW

File: bindings/python/babeltrace/babeltrace/writer.py [hunk 1]
```diff
@@ -179,8 +179,8 @@ def offset_seconds(self):
     @offset_seconds.setter
     def offset_seconds(self, offset_s):
         try:
-            self._clock.offset = bt2.ClockClassOffet(offset_s,
-                                                     self._clock.offset.cycles)
+            self._clock.offset = bt2.ClockClassOffset(offset_s,
+                    self._clock.offset.cycles)
```
Comment (eepp) [inline bindings/python/babeltrace/babeltrace/writer.py]: Align with first argument please!

File: bindings/python/babeltrace/babeltrace/writer.py [hunk 2]
```diff
@@ -203,7 +203,7 @@ def offset(self):
     @offset.setter
     def offset(self, offset):
         try:
-            self._clock.offset = bt2.ClockClassOffet(
+            self._clock.offset = bt2.ClockClassOffset(
                 self._clock.offset.seconds, offset)
```
Comment (eepp) [inline bindings/python/babeltrace/babeltrace/writer.py]: `self._clock.offset.seconds` after `(`, and align `offset` with it.

**Comment by frdeso** [comment]
Made the requested changes.

**Comment by jgalar** [comment]
Merged, thanks!

**Pull request closed without merging**